packages/suite/src/views/wallet/tokens/common/TokensList/TokenRow.tsx

```diff
@@ -115,7 +115,9 @@ const getTokenExplorerUrl = (network: Network, token: EnhancedTokenInfo) => {
 
     const contractAddress = network.networkType === 'cardano' ? token.fingerprint : token.contract;
 
-    return `${explorerUrl}${contractAddress}${network.explorer.queryString}`;
+    const queryString = network.explorer.queryString || '';
```

Comment:
Tiny nit :nerd_face: 
I think `??` is more suitable, because we expect only `undefined`, not falsy values in general.
```suggestion
    const queryString = network.explorer.queryString ?? '';
```

Reply:
be6e52eed0335c23917d1842b2d96afcad787d45